nixpkgs-pinned.nix

```diff
@@ -1,4 +1,4 @@
 fetchTarball {
-  url = "https://github.com/NixOS/nixpkgs/archive/b0bbacb52134a7e731e549f4c0a7a2a39ca6b481.tar.gz";
-  sha256 = "15ix4spjpdm6wni28camzjsmhz0gzk3cxhpsk035952plwdxhb67";
+  url = "https://github.com/NixOS/nixpkgs/archive/7e9b0dff974c89e070da1ad85713ff3c20b0ca97.tar.gz";
+  sha256 = "1ckzhh24mgz6jd1xhfgx0i9mijk6xjqxwsshnvq789xsavrmsc36";
```

Comment:
I know this is scope creep but could you implement `niv` so that the next time this needs to happen it's a brain-dead task?

Reply:
Yeah, @fatho wants to get rid of `fakeroot` so I can actually work on the issue a bit more - why not

Reply:
Done in d6e5eba

Reply:
But I will ask the expert: @FPtje could you take a look at d6e5eba - I added `niv` to `hoff`